zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -559,7 +559,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/UserTaskSearchQueryResponse"
+                $ref: "#/components/schemas/UserTaskSearchQueryResult"
```

Comment:
🔧 I believe we missed the user task search **request** here; it's only number-based and string is not the default.

Reply:
Yes, the `/user-tasks/search` is indeed missing.

Reply:
As far as I can see, it is the only one, though.

Reply:
I couldn't find anything else either. 🤞

Reply:
Do we need a fix for 8.6 as well, adding the alternative media types with number as default?

Reply:
Yes, but good that you say that there number is default. First I just cherry picked the same commit: https://github.com/camunda/camunda/pull/27349
It's confusing... 😄